configurations/s02_pipeline_configuration.py

```diff
@@ -371,7 +383,29 @@
                       )
                     ),
                 ]
-            )
+            ),
+            AnalysisDatasetConfiguration(
+                engagement_db_datasets=["disability", "disabled"],
+                dataset_type=DatasetTypes.DEMOGRAPHIC,
+                raw_dataset="disability_raw",
+                coding_configs=[
+                    CodingConfiguration(
+                        code_scheme=load_code_scheme("disabled"),
```

Comment:
I agree that disability is a better dataset name than disabled, so I think we should rename the code scheme to disability too